fractal.html

```diff
@@ -7,50 +8,55 @@
 </head>
 
 <body>
-<div id="config">
-    <div class="control">
-        <div title="Zoom in">
-            <div>
-                <img id="zoomIn" src="https://maps.gstatic.com/mapfiles/api-3/images/tmapctrl.png" draggable="false">
+    <div id="config">
+        <div class="control">
+            <div title="Zoom in">
+                <div>
+                    <img id="zoomIn" src="https://maps.gstatic.com/mapfiles/api-3/images/tmapctrl.png" draggable="false">
+                </div>
             </div>
-        </div>
-        <div title="Zoom out" id="zoomOut">
-            <div>
-                <img id="zoomOut" src="https://maps.gstatic.com/mapfiles/api-3/images/tmapctrl.png" draggable="false">
+            <div title="Zoom out" id="zoomOut">
+                <div>
+                    <img id="zoomOut" src="https://maps.gstatic.com/mapfiles/api-3/images/tmapctrl.png" draggable="false">
+                </div>
             </div>
         </div>
-    </div>
-    <!--
+        <!--
     <button class="zoom" id="scaleIn">Scale In + </button>
     <button class="zoom" id="scaleOut">Scale Out - </button>
     -->
-</div>
+    </div>
 
-<canvas id="canvas" width="800" height="400"></canvas>
+    <canvas id="canvas" width="800" height="400"></canvas>
 
 
-<script src="triangle.js"></script>
-<script>
-    var canvas = document.getElementById("canvas");
-    var triangle = new Triangle(canvas, 100);
-    triangle.draw();
+    <script src="triangle.js"></script>
+    <script>
+        var canvas = document.getElementById("canvas");
+        var triangle = new Triangle(canvas);
+        triangle.draw(100);
```

Comment:
what does all these hardcoded number mean?  Give all of them more sensible names and make them as constant